docs/build/js-client/3-setting-up.md

```diff
@@ -16,17 +16,15 @@ In newly created `package.json` file add module entry
 }
 ```
 
-We will need JS-client itself
+We will need JS client itself
```

Comment:
maybe for consistency call it `js-client` everywhere just like the npm package?

Reply:
I chose `JS client`